lib/libzutil/zutil_import.c

```diff
@@ -1005,26 +1005,32 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 	}
 
 	if (lio_listio(LIO_WAIT, aiocbps, VDEV_LABELS, NULL) != 0) {
+		/*
+		 * A portion of the requests may have
+		 * been submitted. Clean them up.
+		 */
 		int saved_errno = errno;
+		boolean_t do_slow = 0;
```

Comment:
if it's a boolean_t then you probably want to assign B_FALSE and B_TRUE as well, rather than 0/1

Reply:
Very good point, done.